baker/algolia/indexToAlgolia.ts

```diff
@@ -47,8 +51,12 @@ const computeScore = (record: Omit<PageRecord, "score">): number => {
 }
 
 const getPagesRecords = async () => {
```

Comment:
I feel like this would be nicer with more separation between Gdocs and WP.
I realize that right now the concept of "has been succeeded by Gdoc" is complicating this, but something like this should work:
```ts
const pageviews = await Pageview.getViewsByUrlObj()
const postsApi = await wpdb.getPosts()
const gdocs = await Gdoc.getPublishedGdocs()

// Remove WP posts that have been succeeded by a Google Doc
const wpPosts = postsApi.filter((post) => !gdocs.find((gdoc) => gdoc.slug === post.slug))

const records = [
	...getWpRecords(wpPosts, pageviews),
	...getGdocsRecords(gdocs, pageviews),
]
return records
```

Reply:
Yeah, fair 😅 

I lifted everything into its own function and added some error logging in case the job ever fails.